tests/disko-install/default.nix

```diff
@@ -21,6 +21,7 @@ pkgs.nixosTest {
   nodes.machine = {
     virtualisation.emptyDiskImages = [ 4096 ];
     virtualisation.memorySize = 3000;
+    virtualisation.cores = 8;
```

Comment:
Why? I feel like this might not be a good idea, 8 threads are not as common as 3GB of RAM.

Reply:
That was me testing how qemu + virtio + xcp scales with threads. Not so much so far.